package/utils/sysupgrade-config-verify/files/75_config_restore_verify

```diff
@@ -0,0 +1,18 @@
+. /etc/os-release
+
+config_restore_verify() {
+	cd /
+
+	echo $UPGRADE_META > ${KEEP_DIR}/sysupgrade-config-verify
```

Comment:
Instead of overriding this file on every boot it's better to create this file in the rootfs from the beginning.

Reply:
I did this in order to use a variable UPGRADE_META

coming from a desire to reduce possible code maintenance: if the path of the metadata were to change in Openwrt core, then this package would not require changes